scripts/installer.py

```diff
@@ -144,6 +144,12 @@ def generateCACertificate(self):
             self.printErrorAndExit(f"Error while deleting mkcert binary: {err}")
         logging.info("CA certificate successfully generated")
 
+    def _moveFile(self, file, destination):
+        shutil.move(file, destination)
+    
+    def _deleteFile(self, file):
+        os.remove(thyra_server_path)
```

Comment:
warning, `file` parameter is not used.

don't we have a linter that can spot that before pushing or in the CI?